### ISSUE TITLE
Consider dropping the entire section on static checking (including 'sealed')

### ISSUE THREAD

**Issue by gvanrossum** [gvanrossum/patma#150]
[This section](https://www.python.org/dev/peps/pep-0622/#static-checkers-specification) caused mostly confusion and bewilderment, and I don't consider it essential for the PEP. Static checkers have always managed to figure out how best to check existing Python constructs without formal specifications.

**Comment by Tobias-Kohn** [comment]
Agreed.  Perhaps this is also something that might best be put into a separate PEP if we want to follow up on this.

**Comment by dmoisset** [comment]
+1

**Comment by willingc** [comment]
+1 to simplify and use a future PEP if desired.

**Comment by brandtbucher** [comment]
Agreed. I believe we just inherited it without modification from Ivan's first draft.

**Comment by gvanrossum** [comment]
Done.
